### PR TITLE
MSPSDS-262 update case page design

### PR DESCRIPTION
**Pull request by AnthonyPengelly**
Next I plan to pick up:
* Manage products page
* Manage businesses page
* Page to list all activities you can do

### CODE REVIEW

File: app/controllers/activities_controller.rb [hunk 1]
```diff
@@ -28,7 +28,10 @@ def edit; end
   def create
     respond_to do |format|
       if @activity.save
-        format.html { redirect_to @activity, notice: "Activity was successfully created." }
+        format.html do
+          redirect_to investigation_activities_path(@activity.investigation),
```
Comment (broder) [inline app/controllers/activities_controller.rb]: Do we need the `show` route anymore if we're not redirecting to it here?

File: app/assets/stylesheets/shared.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.aspect-ratio-3-2-container {
+  position: relative;
+  width: 100%;
+  padding-top: 66.66%; /* 3:2 Aspect Ratio */
+
+  .aspect-ratio-contents {
```
[the remark this answers is not in the document]
Comment (AnthonyPengelly) [reply app/assets/stylesheets/shared.scss]: Alongisde the `.product-image` stuff below, I think this is working reasonbly well. It doesn't stretch, it just doesn't always go up to the edges of the container.

File: app/controllers/activities_controller.rb (hunk 1 above)
Comment (AnthonyPengelly) [reply app/controllers/activities_controller.rb]: I think you're right, we don't need the show route. We also don't really need the edit, update or destroy routes either? I wasn't sure whether nuking them all was a good idea at this stage, but shall I just go for it?

File: app/assets/stylesheets/shared.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (broder) [reply app/assets/stylesheets/shared.scss]: Cool!

File: app/controllers/activities_controller.rb (hunk 1 above)
Comment (broder) [reply app/controllers/activities_controller.rb]: Yeah, sounds good

**Pull request merged**